docs/getting-started/user-roadmap-zowe-explorer.md

```diff
@@ -1,121 +1,61 @@
 
-:::tip
+For a better understanding of Zowe Explorer, review the various reference materials that document the VS Code extension.
 
-To identify the resources most relevant for you, use the following definitions of Zowe Explorer skill levels.
+## About Zowe Explorer
 
-* Beginner: You're starting out and want to learn the fundamentals.
-* Intermediate: You have some experience but want to learn more in-depth skills.
-* Advanced: You have lots of experience and are looking to learn about specialized topics.
-:::
+- Check out [**Zowe release notes**](../whats-new/release-notes/release-notes-overview.md) to learn about the latest Zowe Explorer updates.
 
-## Fundamentals
-
-> Zowe skill level: Beginner
-
-* [**Zowe Explorer overview**](../user-guide/ze-install.md)
-
-   New to Zowe Explorer? This overview topic introduces the key features, main components, and benefits of Zowe Explorer.
-
-* [**Zowe Explorer FAQs**](../getting-started/zowe_faq.md#zowe-explorer-faq)
-
-   If you have a question, review the FAQ, which answers the most commonly asked questions about Zowe Explorer.
-
-* [**Blog: Visual Studio Code for Mainframe Via the Zowe Explorer Extension**](https://medium.com/zowe/visual-studio-code-for-mainframe-via-the-zowe-explorer-extension-b679054ffaf7)
-
-   This Medium article outlines the basics of Zowe Explorer, including Getting Started videos.
+- If you have a question, review the [**Zowe Explorer FAQs**](../getting-started/zowe_faq.md#zowe-explorer-faq), which answer the most commonly asked questions about Zowe Explorer.
 
 ## Installing and configuring
 
-> Zowe skill level: Beginner
-
-* [**Installing Zowe Explorer**](../user-guide/ze-install.md#installing-zowe-explorer)
-
-   This page includes the system requirements and steps for installing the Zowe Explorer.
-
-* [**Video: Getting started with Zowe Explorer (Part 1)**](https://youtu.be/G_WCsFZIWt4)
-* [**Video: Getting started with Zowe Explorer (Part 2)**](https://youtu.be/X4oSHrI4oN4)
-
-   These videos help you to get started with Zowe Explorer and show the basic data set use cases.
-
-* [**Zowe Explorer Profiles**](../user-guide/ze-profiles.md)
-
-   This page describes how to create and work with Zowe Explorer profiles. Having a profile enables you to use all functions of the extension, activate the Secure Credential Store plug-in to securely store credentials, and more.
-
-## Using Zowe Explorer
+- Review the system requirements and steps for installing the Zowe Explorer in [**Installing Zowe Explorer**](../user-guide/ze-install.md).
 
-> Zowe skill level: Intermediate
+- Learn how to create and work with [**Zowe Explorer profiles**](../user-guide/ze-profiles.md). A profile enables you to use all functions of the extension, integrate with API Mediation Layer **[correct?]**, and more.
```

Comment:
@adam-wolfe please respond

Reply:
I might change this to something like, "A profile enables you to connect to services running on a mainframe, integrate with the API Mediation Layer, and more". Some profiles give you access to different things, like a user might have a z/osmf profile, an ftp profile, and an ssh profile that all communicate with the mainframe in different ways.